lib/polymer_interop.dart

```diff
@@ -20,98 +20,236 @@ class Polymer {
+  /// `this.children` instead.
+  List<Node> get childNodes => _proxy['childNodes'];
 
+  /// Read-only property that returns a live `List` of the child elements of
+  /// `this`. The list doesn't include text nodes and comments.
   List<Element> get children => _proxy['children'];
 
-  List<Node> get childNodes => _proxy['childNodes'];
-  void set childNodes(List<Node> nodes) {
-    _proxy['childNodes'] = new JsObject.jsify(nodes);
-  }
-
-  Node get parentNode => _proxy['parentNode'];
-  void set parentNode(Node parent) {
-    _proxy['parentNode'] = new JsObject.fromBrowserObject(parent);
-  }
+  // TODO: add docs
+  PolymerClassList get classList => new PolymerClassList(this);
 
+  /// Read-only property that returns the node's first child in the tree, or
+  /// null if the node is childless.
   Node get firstChild => _proxy['firstChild'];
-  void set firstChild(Node child) {
-    _proxy['firstChild'] = new JsObject.fromBrowserObject(child);
-  }
 
-  Node get lastChild => _proxy['lastChild'];
-  void set lastChild(Node child) {
-    _proxy['lastChild'] = new JsObject.fromBrowserObject(child);
-  }
+  /// Read-only property that returns the object's first child Element, or null
+  /// if there are no child elements.
+  HtmlElement get firstElementChild => _proxy['firstElementChild'];
 
-  Element get firstElementChild => _proxy['firstElementChild'];
-  void set firstElementChild(Element child) {
-    _proxy['firstElementChild'] = new JsObject.fromBrowserObject(child);
-  }
+  /// Sometimes it’s necessary to access the elements which have been distributed
+  /// to a given <content> insertion point or to know to which <content> a given
+  /// node has been distributed. The getDistributedNodes and
+  /// getDestinationInsertionPoints methods, respectively, provide this information.
+  List<Node> getDestinationInsertionPoints() =>
+      _proxy.callMethod('getDestinationInsertionPoints');
 
-  Element get lastElementChild => _proxy['lastElementChild'];
-  void set lastElementChild(Element child) {
-    _proxy['lastElementChild'] = child;
-  }
+  /// Sometimes it’s necessary to access the elements which have been distributed
+  /// to a given <content> insertion point or to know to which <content> a given
+  /// node has been distributed. The getDistributedNodes and
+  /// getDestinationInsertionPoints methods, respectively, provide this information.
+  List<Node> getDistributedNodes() => _proxy.callMethod('getDistributedNodes');
 
-  Node get previousSibling => _proxy['previousSibling'];
-  void set previousSibling(Node sibling) {
-    _proxy['previousSibling'] = new JsObject.fromBrowserObject(sibling);
+  // TODO: find out what this does
+  getOwnerRoot() => _proxy.callMethod('getOwnerRoot');
+
+  /// Sets or gets the HTML syntax describing the element's descendants.
+  String get innerHtml => _proxy['innerHTML'];
+  /// Sets or gets the HTML syntax describing the element's descendants.
```

Comment:
Heh, is it not documented in the polymer js code? I think it is fine to leave the comment as is for now in that case :).